### PR TITLE
Replace accessible with available

### PR DESCRIPTION
**Pull request by sandandsnow**
Addressing change requested in Issue 48

### CODE REVIEW

**Comment by sandandsnow** [comment]
@npdoty please review when you have time

**Comment by samuelweiler** [comment]
To fix #48

File: index.html [hunk 1]
```diff
@@ -274,7 +274,7 @@ <h2>Feasibility</h2>
           <h2>Fingerprinting mitigation levels of success</h2>
           <p>There are different levels of success in mitigating browser fingerprinting:</p>
           <dl>
-          <dt>Decreased fingerprinting surface</dt><dd>Removing the source of entropy or accessible attributes that can be used for fingerprinting.</dd>
+          <dt>Decreased fingerprinting surface</dt><dd>Removing the source of entropy or visible attributes that can be used for fingerprinting.</dd>
```
Comment (npdoty) [inline index.html]: Maybe this should be "available" too, as "visible" might imply visibility to the eye (like features that are hidden or visible via CSS).

Comment (sandandsnow) [reply index.html]: Fixed

**Pull request merged**